src/item/ItemComponentsTrait.php

```diff
@@ -60,7 +78,48 @@ public function getComponents(): CompoundTag {
 			}
 			$components->setTag($component->getName(), $tag);
 		}
+
+
+
+		if ($this instanceof Sword) {
+            $properties->setString('enchantable_slot', 'sword');
+            $properties->setInt('enchantable_value', $this->getEnchantability());
+        }
+
+        if ($this instanceof Pickaxe) {
+            $properties->setString('enchantable_slot', 'pickaxe');
+            $properties->setInt('enchantable_value', $this->getEnchantability());
+        }
+
+        if ($this instanceof Axe) {
+            $properties->setString('enchantable_slot', 'axe');
+            $properties->setInt('enchantable_value', $this->getEnchantability());
+        }
+
+
+        if ($this instanceof Hoe) {
+            $properties->setString('enchantable_slot', 'hoe');
+            $properties->setInt('enchantable_value', $this->getEnchantability());
+        }
+
+
+        if ($this instanceof Shovel) {
+            $properties->setString('enchantable_slot', 'shovel');
+            $properties->setInt('enchantable_value', $this->getEnchantability());
+        }
```

Comment:
You should use a match case in enchantable_slot for the value. For the rest it's similar in every case so...

Reply:
And you should use " instead of '